src/execution/vm/bytecode_generator.cpp

```diff
@@ -2054,6 +2054,22 @@ void BytecodeGenerator::VisitBuiltinStringCall(ast::CallExpr *call, ast::Builtin
   }
 }
 
+void BytecodeGenerator::VisitBuiltinArithCall(ast::CallExpr *call, ast::Builtin builtin) {
+  LocalVar exec_ctx = VisitExpressionForRValue(call->Arguments()[0]);
+  LocalVar first_input = VisitExpressionForRValue(call->Arguments()[1]);
+  LocalVar second_input = VisitExpressionForRValue(call->Arguments()[2]);
+  LocalVar ret = ExecutionResult()->GetOrCreateDestination(call->GetType());
+  switch (builtin) {
+	//case ast::Builtin::IntMod:
```

Comment:
@kjobanputra Is there a reason this and the `IntMod` reference below were left commented out?

Reply:
Probably dead code. I took it out. Thanks!